### PR TITLE
E45fd new UI form editor

### PR DESCRIPTION
**Pull request by JhumanJ**

### CODE REVIEW

File: resources/js/components/open/forms/components/form-components/FormCustomCode.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
-  <collapse class="p-5 w-full border-b" :default-value="false">
+  <collapse class="py-3 w-full border-b" :default-value="false">
     <template #title>
       <h3 class="font-semibold text-lg">
-        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-gray-500 mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-nt-blue mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```
Comment (JhumanJ) [inline resources/js/components/open/forms/components/form-components/FormCustomCode.vue:5]: `text-nt-blue` should not exist anymore

File: resources/js/components/open/forms/components/FormEditor.vue [hunk 2]
```diff
@@ -2,11 +2,27 @@
...
     <!-- Form fields selection -->
     <v-tour name="tutorial" :steps="steps" />
-    <div class="w-full md:w-1/2 lg:w-2/5 border-r relative overflow-y-scroll md:max-w-sm flex-shrink-0">
+    <div class="w-full md:w-1/2 lg:w-2/5 border-r relative overflow-y-scroll md:max-w-sm flex-shrink-0 p-4">
       <div class="p-5 bg-blue-50 border-b text-nt-blue-dark md:hidden">
         We suggest you create this form on a device with a larger screen such as computed. That will allow you
         to preview your form changes.
       </div>
+      
+      <a href="javascript:;" @click="$router.back()" class="flex text-blue mb-5">
```
Comment (JhumanJ) [inline resources/js/components/open/forms/components/FormEditor.vue:11]: what's this `href="javascript:;"` ?

File: resources/js/components/open/forms/components/FormEditor.vue [hunk 3]
```diff
@@ -2,11 +2,27 @@
...
+        <svg class="w-4 h-4 text-blue mt-1" viewBox="0 0 6 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M5 9L1 5L5 1" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+        </svg>
+        Go back
+      </a>
+      <h3 class="font-semibold text-lg">{{form.title}}</h3>
+      <small>Edited 2 months ago</small>
```
Comment (JhumanJ) [inline resources/js/components/open/forms/components/FormEditor.vue:18]: todo: replace with actual date (can be passed in the FormResource in the back-end as `last_edited_human`). Only show for edit, not create obviously

File: resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <template>
-  <collapse class="p-5 w-full" :default-value="true">
+  <collapse class="py-3 w-full border-b" :default-value="true">
     <template #title>
       <h3 class="font-semibold text-lg relative">
-        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-gray-500 mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-nt-blue mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```
Comment (JhumanJ) [inline resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue:5]: no `nt-blue`   
`nt` is for "notion-tools", the old Name of NotionForms

**Review by JhumanJ — COMMENTED** [review]
@chiragnotionforms here are some comments on this

File: resources/js/components/open/forms/components/form-components/FormCustomCode.vue (hunk 1 above)
Comment (chiragnotionforms) [reply resources/js/components/open/forms/components/form-components/FormCustomCode.vue:5]: image

Class is valid and working also

File: resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue (hunk 4 above)
Comment (chiragnotionforms) [reply resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue:5]: we still not removed from class name so it's working

File: resources/js/components/open/forms/components/FormEditor.vue (hunk 2 above)
Comment (JhumanJ) [reply resources/js/components/open/forms/components/FormEditor.vue:11]: You need to use `href="#"` instead, and then use `@click.prevent`

File: resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue (hunk 4 above)
Comment (JhumanJ) [reply resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue:5]: Yes but let's not use anymore, anyway this colors isn't used in figma

File: resources/js/components/open/forms/components/FormEditor.vue (hunk 2 above)
Comment (chiragnotionforms) [reply resources/js/components/open/forms/components/FormEditor.vue:11]: ok

File: resources/js/components/open/forms/components/FormEditor.vue (hunk 3 above)
Comment (chiragnotionforms) [reply resources/js/components/open/forms/components/FormEditor.vue]: Done

**Pull request merged**